zppy/templates/ts.bash

```diff
@@ -65,6 +65,7 @@ cat input.txt | ncclimo \
 -c {{ case }} \
 -v {{ vars }} \
 --mem_mb=0 \
+--split \
```

Comment:
Adding `--split`, as in https://github.com/E3SM-Project/e3sm_diags/pull/593.

Reply:
@chengzhuzhang @czender Even with this change, I'm getting the following. I wasn't running into any issues with the previous Unified's version of NCO.
```
$ tail -n 11 ts_land_monthly_1850-1851-0002.o190591
ncremap: ERROR Failed to horizontally regrid. cmd_rgr[0] failed. Debug this:
 ncks -O -t 1 --no_tmp_fl --hdr_pad=10000 --no_tmp_fl --hdr_pad=10000 --gaa remap_script=ncremap --gaa remap_hostname=chr-0493 --gaa remap_version=5.0.7    --rgr lat_nm_out=lat --rgr lon_nm_out=lon --rgr no_stagger --rgr sgs_frc_nm=landfrac#sgs_msk_nm=landmask --map_fl="/home/ac.zender/data/maps/map_ne30pg2_to_cmip6_180x360_aave.20200201.nc" "trash/FSH_185001_185112.nc" "output/FSH_185001_185112.nc"
ncclimo: ERROR Failed to regrid. cmd_rgr[0] failed. Debug this:
 ncremap -u .pid602674.split.0.tmp --nco_opt="--no_tmp_fl --hdr_pad=10000" -t 1  --no_stagger -P sgs  --map=/home/ac.zender/data/maps/map_ne30pg2_to_cmip6_180x360_aave.20200201.nc trash/FSH_185001_185112.nc output/FSH_185001_185112.nc
/lcrc/soft/climate/e3sm-unified/spack/spack_for_mache_1.4.1/var/spack/environments/e3sm_unified_1_7_0rc1_chrysalis_gnu_openmpi/.spack-env/view/bin/ncclimo: line 2836: kill: (602753) - No such process
/lcrc/soft/climate/e3sm-unified/spack/spack_for_mache_1.4.1/var/spack/environments/e3sm_unified_1_7_0rc1_chrysalis_gnu_openmpi/.spack-env/view/bin/ncclimo: line 2836: kill: (602754) - No such process
+ '[' 1 '!=' 0 ']'
+ cd /lcrc/group/e3sm/ac.forsyth2/zppy_test_complete_run_output/v2.LR.historical_0201/post/scripts
+ echo 'ERROR (1)'
+ exit 1
ncks: ERROR nco_rgr_wgt() (aka "the regridder") reports unable to find sgs_frc_nm = landfrac in current input file, and unable to identify filename (ending with slash '/' or backslash '\', depending on the operating system) portion of that string to serve as local external file for sgs_frc input, exiting
```

Reply:
My interpretation of this error comes from the error message itself. It appears that the input file `trash/FSH_185001_185112.nc` does not contain the `landfrac` variable that `ncremap` needs to regrid ELM files correctly. There are two routes you can take: either make sure the workflow retains `landfrac` in each input file until `ncremap` is invoked, or, alternatively use the `--sgs_frc=/path_to_file_with_landfrac/filename/landfrac` to point `ncremap` to a valid `landfrac` in a second file. Does this explain the problem you encountered? If not, please provide full commands and leave the inputs where I can find them.